src/services/comment.service.js

```diff
@@ -13,6 +13,26 @@ class CommentService {
   static createComment(newComment) {
     return Comments.create(newComment);
   }
+
+  /**
+   * @param {number} property
+   * @returns {find} this function finds comment
+  */
+  static findCommentByProperty(property) {
+    return Comments.findOne({
+      where: property
+    });
+  }
+
+  /**
+   * @param {number} property
+   * @returns {delete} this function deletes comment
+  */
+  static deleteCommentByProperty(property) {
+    return Comments.destroy({
```

Comment:
The does not meet the acceptance criteria. You should use softDelete instead.

`- The comment is not deleted from the backend but is no longer available on the front end`

Reply:
Okay let me use it